packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -427,8 +427,18 @@ export const Webchat = forwardRef((props, ref) => {
   }, [])
 
   const coverComponent = () => {
+    const coverComponentProps = getThemeProperty(
+      'coverComponent.props',
+      props.coverComponent && props.coverComponent.props
+    )
+
     if (CoverComponent && webchatState.isCoverComponentOpen)
-      return <CoverComponent closeComponent={closeCoverComponent} />
+      return (
+        <CoverComponent
+          closeComponent={closeCoverComponent}
+          coverComponentProps={coverComponentProps}
```

Comment:
I think it would be better if the attributes of `coverComponentProps` are sent directly to the `coverComponent` instead of sending them encapusalted with another props (`coverComponentProps`)
```suggestion
          {...coverComponentProps}
```